base/exports.jl

```diff
@@ -1149,6 +1149,7 @@ export
     ntoh,
     open,
     pipe,
+    Pipe,
```

Comment:
I think it might just have to be `Pipe` and `pipeline`, though I sorta dread having to tell people they have to go from one character to 8 for this functionality. 

Of course, I also suggested `const 🚰= Pipe`